### ISSUE TITLE
WCAG contrast ratios don't seem to be right

### ISSUE THREAD

**Issue by clairedenning** [saneef/color-color#33]
Ratio values are very similar between white and black on each swatch.

<img width="227" height="1830" alt="Image" src="https://github.com/user-attachments/assets/b2e856d8-c630-4dcd-8af4-b728dd305136" />
